docker-images/kafka-connect/scripts/kafka_connect_config_generator.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+CONFIG_FILE=$1
+
+# Write the config file
+cat > ${CONFIG_FILE:-/tmp/strimzi-connect.properties} <<EOF
```

Comment:
I think it would be ever-so-slightly simpler if this script just wrote the config to stdout, and left it to `kafka_connect_run.sh` to redirect that to a file. That way you don't have to worry about `$1` not being set. That way, when you call it in the run script instead of writing to file and then `cat`ing the file you just wrote you could `tee`

```bash
# Generate the config file
echo "Starting Kafka connect with configuration:"
./kafka_connect_config_generator.sh | tee /tmp/strimzi-connect.properties
```